src/applications/my-education-benefits/containers/App.jsx

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
+import Breadcrumbs from '@department-of-veterans-affairs/component-library/Breadcrumbs';
 
 import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
 import formConfig from '../config/form';
 
 export default function App({ location, children }) {
   return (
-    <RoutedSavableApp formConfig={formConfig} currentLocation={location}>
-      {children}
-    </RoutedSavableApp>
+    <>
+      <Breadcrumbs>
```

Comment:
Can we fill these out with the href values?

Reply:
I'm not entirely sure if we have these values at the moment.

Reply:
(syncd in slack)